services/movie.py

```diff
@@ -28,13 +33,15 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
The type annotations for `genres_ids` and `actors_ids` should be `Optional[list]` to indicate that these parameters can be `None`.